### ISSUE TITLE
When convert from keras to tflite, the output is different from what it shoud be.

### ISSUE THREAD

**Issue by wbjnpu** [tensorflow/tensorflow#62833]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installation (pip package or built from source):
- TensorFlow library (version, if pip package or github SHA, if built from source):

### 2. Code

Provide code to help us reproduce your issues using one of the following options:

#### Option A: Reference colab notebooks

1)  Reference [TensorFlow Model Colab](https://colab.research.google.com/gist/ymodak/e96a4270b953201d5362c61c1e8b78aa/tensorflow-datasets.ipynb?authuser=1): Demonstrate how to build your TF model.
2)  Reference [TensorFlow Lite Model Colab](https://colab.research.google.com/gist/ymodak/0dfeb28255e189c5c48d9093f296e9a8/tensorflow-lite-debugger-colab.ipynb): Demonstrate how to convert your TF model to a TF Lite model (with quantization, if used) and run TFLite Inference (if possible).

```
(You can paste links or attach files by dragging & dropping them below)
- Provide links to your updated versions of the above two colab notebooks.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

#### Option B: Paste your code here or provide a link to a custom end-to-end colab

```
(You can paste links or attach files by dragging & dropping them below)
- Include code to invoke the TFLite Converter Python API and the errors.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

https://github.com/Z-yq/TensorflowASR/blob/v2/asr/models/chunk_conformer_blocks.py this file CTCdecoder is not correct,but converted tflite of conformerblock work well, and I convert it using https://github.com/TensorSpeech/TensorFlowASR . And I have control input shape when load weight. It complex me a long time. maybe because its dynamic train?
### 3. Failure after conversion
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results and/or has lesser accuracy.
- Model produces correct results, but it is slower than expected.

**Comment by wbjnpu** [comment]
my problem is that "Model produces wrong results and/or has lesser accuracy"

**Comment by wbjnpu** [comment]
And I find that .h5 infer is correct， but tflite is wrong. they are the same size and without any quant.

**Comment by LakshmiKalaKadali** [comment]
Hi @pkgoogle,

I reproduced the code and noticed that each tflite quantized file has the size 105528 bytes like the model. Please look into the issue.
Thank You

**Comment by pkgoogle** [comment]
Hi @wbjnpu, can you please upload your .h5 and .tflite file and let us know the inference value you expected vs what you saw? and how you know that the .tflite version is incorrect? Thanks for your help! Generally the more information you provide the faster we can help you.
